src/actions/tabs.js

```diff
@@ -30,11 +30,17 @@ export function updateTab(url: string, framework: string): Action {
   };
 }
 
-export function addTab(url: string, framework?: string): Action {
+export function addTab(
+  url: string,
+  framework?: string,
+  isOriginal?: boolean
+): Action {
+  console.log("ADD_TAB", url, framework, isOriginal);
   return {
     type: "ADD_TAB",
     url,
-    framework
+    framework,
+    isOriginal
   };
```

Comment:
we might want to change this API to `addTab(source)`, `closeTab(source)` and do everything internally in the action

Reply:
Done!